src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class names 'calendar--start-day-sun' and 'calendar--month-length-31' suggest that you might be trying to modify the calendar's behavior with modifiers. Ensure that these modifiers are used correctly according to the BEM methodology and that they are implemented in the SCSS files.